src/layer/LayerGroup.js

```diff
@@ -147,6 +148,23 @@ export const LayerGroup = Layer.extend({
 	// Returns the internal ID for a layer
 	getLayerId(layer) {
 		return Util.stamp(layer);
+	},
+
+	// @method getBounds(): LatLngBounds
+	// Returns the LatLngBounds of the Layer Group (created from bounds and coordinates of its children).
+	getBounds(visitedIds = new Set()) {

```

Comment:
Let us make the variable `visitedIds` "private":
```suggestion
	getBounds(_visitedIds = new Set()) {

```